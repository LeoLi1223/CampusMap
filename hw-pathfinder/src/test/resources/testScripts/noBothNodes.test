# tests that the both nodes don't exists in the graph

CreateGraph graph1

AddNode graph1 B
AddNode graph1 C

AddEdge graph1 B C 10.0

FindPath graph1 A D