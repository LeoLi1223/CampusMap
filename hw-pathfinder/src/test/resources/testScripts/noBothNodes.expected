# tests that the both nodes don't exists in the graph

created graph graph1

added node B to graph1
added node C to graph1

added edge 10.000 from B to C in graph1

unknown: A
unknown: D